dumpling/tests/composite_string_key/run.sh

```
@@ -0,0 +1,86 @@
+
+# Run dumpling with --rows parameter to force chunking
+# With --rows 5, tables will be split into multiple chunks
+# Each chunk should contain a complete INSERT statement with ~5 rows
+run_dumpling --rows 5
+
+for file_path in "$DUMPLING_BASE_NAME"/data/*; do
+  base_name=$(basename "$file_path")
+  table_name="${base_name%.sql}"
+  
+  # Count the number of chunk files created
+  chunk_count=$(ls -1 "$DUMPLING_OUTPUT_DIR"/composite_string_key.$table_name.*.sql 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)